source/github-helpers/github-url.ts

```diff
@@ -49,13 +21,51 @@ export default class GitHubURL extends URL {
 		return this;
 	}
 
+	toString() {
+		return this.href;
+	}
+
+	private disambiguateReference(ambiguousReference: string[]): {branch: string; filePath: string} {
+		const branch = ambiguousReference[0];
+		const filePathFromSearch = this.searchParams.getAll('path[]').join('/');
+		if (filePathFromSearch) {
+			// Clear the search from the url, so it does not get passed to the new url.
```

Comment:
```suggestion
			// Clear the search from the `path[]` param, so it does not get passed to the new url.
```

I dont think that is even needed any more WFYT?

Reply:
Yeah I think it can be dropped, it's self-explanatory: _we used it, so we're dropping it._